libr/core/cmd_anal.c

```diff
@@ -5129,6 +5129,16 @@ static void cmd_anal_aav(RCore *core, const char *input) {
 	seti ("search.align", o_align);
 }
 
+static bool should_aav(RCore *core) {
+	int X_LEN = strlen("x86");
```

Comment:
w t f? use r_str_startswith().

the compiler implements strlen at compile time, so this is a constant value, even if you do strlen("x86" in the parameter it will be replaced by 3 in the final binary, no calls to strlen